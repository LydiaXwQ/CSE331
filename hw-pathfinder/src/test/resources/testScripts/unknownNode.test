# Find a path with Dijkstra's algorithm when
# a given node is not in the graph.

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B

AddEdge simpleGraph A B 10.0

FindPath simpleGraph A B
FindPath simpleGraph B D