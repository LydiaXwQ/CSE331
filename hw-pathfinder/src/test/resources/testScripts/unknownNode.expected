# Find a path with Dijkstra's algorithm when
# a given node is not in the graph.

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph

added edge 10.000 from A to B in simpleGraph

path from A to B:
A to B with weight 10.000
total cost: 10.000
unknown: D